src/backend/joanie/tests/core/api/organizations/test_contracts_signature_link.py

```diff
@@ -0,0 +1,343 @@
+"""
+Test suite for Organization Contracts signature link API endpoint.
+"""
+import random
+from http import HTTPStatus
+
+from django.utils import timezone
+
+from joanie.core import enums, factories
+from joanie.core.models import OrganizationAccess
+from joanie.tests.base import BaseAPITestCase
+
+
+class OrganizationApiContractSignatureLinkTest(BaseAPITestCase):
+    """
+    Test suite for Organization Contracts signature link API endpoint.
+    """
+
+    def test_api_organization_contracts_signature_link_without_owner(self):
+        """
+        Authenticated users which is not an organization owner should not be able
+        to sign contracts in bulk.
+        """
+        user = factories.UserFactory(
+            is_staff=random.choice([True, False]),
+            is_superuser=random.choice([True, False]),
+        )
+        order = factories.OrderFactory(
+            state=enums.ORDER_STATE_VALIDATED,
+            product__contract_definition=factories.ContractDefinitionFactory(),
+        )
+        organization_roles_not_owner = [
+            role[0]
+            for role in OrganizationAccess.ROLE_CHOICES
+            if role[0] != enums.OWNER
+        ]
+        factories.UserOrganizationAccessFactory(
+            user=user,
+            organization=order.organization,
+            role=random.choice(organization_roles_not_owner),
+        )
+
+        order.submit_for_signature(order.owner)
+        order.contract.submitted_for_signature_on = timezone.now()
+        order.contract.student_signed_on = timezone.now()
+        token = self.generate_token_from_user(user)
+
+        response = self.client.get(
+            f"/api/v1.0/organizations/{order.organization.id}/contracts-signature-link/",
+            HTTP_AUTHORIZATION=f"Bearer {token}",
+        )
+
+        self.assertEqual(response.status_code, HTTPStatus.FORBIDDEN)
+        self.assertEqual(
+            response.json(),
+            {"detail": "You do not have permission to perform this action."},
+        )
```

Comment:
```suggestion
        self.assertContains(
            response,
             "You do not have permission to perform this action.",
             status_code=HTTPStatus.FORBIDDEN
        )
```
This might be a shorter way to present those 2 asserts in this test in my opinion, WDYT ?